modules/uploading-an-existing-notebook-file-from-a-git-repository-using-code-server.adoc

```diff
@@ -0,0 +1,30 @@
+:_module-type: PROCEDURE
+
+[id='uploading-an-existing-notebook-file-from-a-git-repository-using-code-server_{context}']
+= Uploading an existing notebook file from a Git repository by using code-server
+
+[role='_abstract']
+You can use the code-server user interface to clone a Git repository into your workspace to continue your work or integrate files from an external project.
+
+.Prerequisites
+* A launched and running code-server workbench.
+* Read access for the Git repository you want to clone.
+
+.Procedure
+. Copy the HTTPS URL for the Git repository.
++
+** On GitHub, click *&#10515; Code* -> *HTTPS* and click the Copy URL to clipboard icon.
+** On GitLab, click *Code* and click the Copy URL icon under *Clone with HTTPS*.
+. In your code-server workbench, from the Activity Bar, select the menu icon (image:images/codeserver-menu-icon.png[The Menu icon]) > *View* > *Command Palette*.
+. In the Command Palette, enter `Git: Clone`, and then select `Git: Clone` from the list.
+. Paste the HTTPS URL of the repository that contains your notebook, and then press Enter.
+. If prompted, enter your username and password for the Git repository.
+. Select a folder to clone the repository into, and then click *OK*.
+. When the repository is cloned, a dialog appears asking if you want to open the cloned repository. Click *Open* in the dialog.
+
+.Verification
+* Check that the contents of the repository are visible in the Explorer in the code-server workbench, or run the *ls* command in the terminal to verify that the repository shows as a directory.
```

Comment:
in the Explorer in the code-server workbench
edit:
in the  in the code-server Explorer

change **ls** to `ls`

Reply:
apparently the full VS Code terminology for the UI is **Explorer view** displayed in the **primary side bar**. Did not know that https://code.visualstudio.com/docs/getstarted/userinterface#_basic-layout